src/Bitmap.cpp

```diff
@@ -24,36 +24,50 @@ static CLSID  encoderClsid;
 #include <sys/stat.h> // for mkdir
 #endif
 
-static unsigned char* bmf, *bmPixels, *bmp;
+namespace CovidSim
+{
+	namespace BitMap
```

Comment:
Bikeshedding comment :-): Can it be `Bitmap` as it is not a `std::map<>` derived type which is what capitalising the M makes me think of!

Reply:
Tried Bitmap initially but it clashes with the class of the same name. I suppose it could be changed to Graphics or some such.